contracts/KashiPair.sol

```diff
@@ -751,4 +751,17 @@ contract KashiPair is ERC20, BoringOwnable, IMasterContract {
         feeTo = newFeeTo;
         emit LogFeeTo(newFeeTo);
     }
+
+    /// @notice Helper function to calculate the collateral shares that are needed for `borrowPart`,
+    /// taking the current exchange rate into account.
+    function getCollateralSharesForBorrowPart(uint256 borrowPart) public view returns (uint256) {
```

Comment:
@Clearwood We can move this one of course :). But might be useful in general.

Reply:
This should be in a helper contract or in helper js, not here.

Reply:
Moved to KashiPairHelper 👍